yvalenta/lib/server/boot.rb

```diff
@@ -0,0 +1,3 @@
+require 'server'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.